#goal definition
float32 AngleToTurn
---
#result definition
bool MotionCompleted
---
#feedback

